blueocean-commons/src/main/java/io/jenkins/blueocean/commons/stapler/export/Property.java

```diff
@@ -141,15 +141,14 @@ public void writeTo(Object object, TreePruner pruner, DataWriter writer) throws
         if (merge) {
             // merged property will get all its properties written here
             if (d != null) {
-                Model model;
-                try {
-                    model = owner.get(d.getClass(), parent.type, name);
-                } catch (NotExportableException e) {
+                if (d.getClass().getAnnotation(ExportedBean.class) == null) {
```

Comment:
Same as above, lets not update stapler code directly here. Please open a PR on stapler.